File: LargestElementArray/LargestElement.js
// Largest Element in the Array: 

// n = parseInt(prompt("Enter number: "))
// arr = prompt("Enter values: ").split(' ').map(Number)

n = 5
arr = [100, 28687, 9879, 9873, 10]

maximum = arr[0]
for (let i = 0; i < n; i++) {
    if (arr[i] > maximum) {
        maximum = arr[i]
    }
}
console.log(`Largest Element in the Array is: ${maximum}`)

File: LargestElementArray/LargestElement.c
// Largest Element in the Array:
#include <stdio.h>
void LargestElement(int n, int arr[])
{
    int maximum = arr[0];
    for (int i = 0; i < n; i++)
    {
        if (arr[i] > maximum)
        {
            maximum = arr[i];
        }
    }
    printf("\nLargest Element in the Array is : %d", maximum);
}
int main()
{
    int i, n;
    printf("Enter number: ");
    scanf("%d", &n);
    int arr[n];
    printf("Enter values: ");
    for (i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }
    LargestElement(n, arr);

    return 0;
}